colors/material.vim

```diff
@@ -189,6 +189,21 @@ call s:HL('jsTemplateExpression', s:red, '', '')
 call s:HL('jsThis', s:red, '', '')
 call s:HL('jsUndefined', s:orange, '', '')
 
+" Golang 
+call s:HL('goComment', s:gray5, '', '')
+call s:HL('goFunctionCall', s:blue, '', 'bold')
```

Comment:
I personally have a preference not to use bold text for syntax highlighting (it's used to highlight bold text in markdown files and for some Vim-specific features).

Reply:
I'll tweak that!